Documentation/dl_build.md

```diff
@@ -18,7 +18,6 @@ To build `etcd` from the `master` branch without a `GOPATH` using the official `
 $ git clone https://github.com/coreos/etcd.git
 $ cd etcd
 $ ./build
-$ ./bin/etcd
```

Comment:
OK, probably remove the run lines from the build examples then and have some comment in `Test the installation` like:
```
Start etcd:
$ ./bin/etcd # $GOPATH/bin/etcd if built using GOPATH
```